startup.py

```diff
@@ -241,4 +241,64 @@ def prepare_launch(self, exec_path, args, file_to_open=None):
 
         :returns: :class:`LaunchInformation` instance
         """
-        return LaunchInformation(exec_path, args)
\ No newline at end of file
+
+        # Get the basic environment variables to launch Nuke. Do this by reusing Toolkit classic's
+        # bootstrap code.
+        startup_python_path = os.path.join(
+            self.disk_location,
+            "python",
+            "startup"
+        )
+
+        with self.temporary_import(startup_python_path, "bootstrap") as bootstrap:
```

Comment:
not sure i understand this part - why do we need to import `python/startup`? Is it because we keep stuff in that file for backwards compatibility with older launch apps that might call it? In that case, could we do it the other way, and have the old legacy bootstrap interface in `python/startup` call `startup.py` (the new official and cleaned up interface) instead of the other way around? This feels like a very complicated approach - would love to discuss if there is a simpler way forward.